ui/src/organizations/actions/orgView.ts

```diff
@@ -35,6 +35,23 @@ export const getTasks = (orgID: string) => async dispatch => {
   dispatch(populateTasks(tasks))
 }
 
+export interface PopulateDashboards {
+  type: ActionTypes.PopulateDashboards
```

Comment:
Should we use the top level dashboards reducer instead to do this? It seems unfortunate that there would be two copies of the same dashboard in Redux (one from the dashboards reducer, and one from the org view reducer).

Reply:
I might agree but wanted to continue with existing pattern, would be easy to switch